Infinion/ZeroPilotSwHooks/SensorFusionIntercept.cpp

```diff
@@ -0,0 +1,175 @@
+#include "SensorFusion.hpp"
+
+#include <iostream>
+#include <fstream>
+#include <stdio.h>
+#include <string>
+#include <cmath>
+
+/***********************************************************************************************************************
+ * Prototypes
+ **********************************************************************************************************************/
+
+constexpr float m_pi = 3.14159;
```

Comment:
Are we gonna do anything about these declarations of Pi anytime soon?

Reply:
yes, I'm gonna treat that as my next task, but I'm gonna be making a dedicated PR for it.